.ci/prepare-settings.sh

```diff
@@ -18,7 +18,7 @@ replace() {
 replace SONATYPE_USER "$SONATYPE_USER"
 replace SONATYPE_PWD "$SONATYPE_PWD"
 replace GPG_PASSPHRASE "$GPG_PASSPHRASE"
-replace GPG_KEY "$GPG_KEYNAME"
+replace GPG_KEYNAME "$GPG_KEYNAME"
```

Comment:
This one was discovered yesterday: https://github.com/checkstyle/checkstyle/issues/12762#issuecomment-1440672537